quipucords/scanner/job.py

```diff
@@ -18,6 +18,30 @@
 logger = logging.getLogger(__name__)
 
 
+def create_task_runner(scan_job: ScanJob, scan_task: ScanTask):
+    """Create ScanTaskRunner using scan_type and source_type."""
+    scan_type = scan_task.scan_type
+    if scan_type == ScanTask.SCAN_TYPE_CONNECT:
+        return create_connect_task_runner(scan_job, scan_task)
+    if scan_type == ScanTask.SCAN_TYPE_INSPECT:
+        return create_inspect_task_runner(scan_job, scan_task)
+    if scan_type == ScanTask.SCAN_TYPE_FINGERPRINT:
+        return FingerprintTaskRunner(scan_job, scan_task)
+    return None
```

Comment:
I know we are just moving, but I'd do some defensive programming here in a subsequent PR changing "return None" to "assert False/raise NotImplementedError" as returning `None` here is clearly a bug.

(You must be noticing the original implementation is pretty much permissive with non sense values 😅 )

Reply:
I support this suggestion!